content/docs/dvclive/get-started.md

```diff
@@ -109,6 +109,20 @@ not.
 
 See `Live.log()`, `Live.log_image()` and `Live.log_plot()` for more details.
 
+### HTML report
+
+If and when `step` is updated, DVCLive generates or updates an HTML report in
+`dvclive/report.html` which will contain all the logged data.
```

Comment:
Is this only true if `report="html"` was passed to `Live()` (per `set/next_step()` refs update)?

If so do we need to specify so [in the Get Started](https://dvc.org/doc/dvclive/get-started#initialize-dvclive) or is it the default?

Reply:
> Is this only true if report="html" was passed... or is it the default?

I see it's the default. No further change needed in GS then 👍🏼